cpp/.clang-format

```diff
@@ -71,8 +71,28 @@ ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
-IncludeBlocks: Preserve
-IncludeIsMainRegex: '([-_](test|unittest))?$'
+IncludeBlocks: Regroup
+IncludeCategories:
+  - Regex:           '^"' # quoted includes
+    Priority:        1
+  - Regex:           '^<(benchmarks|tests)/' # benchmark includes
+    Priority:        2
+  - Regex:           '^<(cuspatial_test|cuproj_test)/' # cuSpatial/cuProj test includes
+    Priority:        3
+  - Regex:           '^<(cuspatial|cuproj)/' # cuSpatial/cuProj includes
+    Priority:        4
+  - Regex:           '^<(cudf_test|cudf|cuml|raft|kvikio)' # Other RAPIDS includes
```

Comment:
I thought about removing cuml/raft/kvikio since they are not used, should it?

Reply:
We could, but I'm also trying to keep things relatively consistent across RAPIDS repos. But I'm not confident in that direction. Thoughts?

Reply:
Yeah I'm not confident either - especially if we apply this remove-not-used rule, then what about the cuda libraries below? If we remove cuda rules, are we sure that future contributors do not use them? 

From that end, I think the future-forward approach is to maintain the disjunction set from all RAPIDS libraries to embed the collective behavioral customs. This reduces possible conflicts when migrating code or merging repos.